src/map/ai/states/magic_state.cpp

```diff
@@ -182,7 +182,7 @@ bool CMagicState::CanCastSpell(CBattleEntity* PTarget)
     {
         return false;
     }
-    if (distance(m_PEntity->loc.p, PTarget->loc.p) > 40)
```

Comment:
The intention is to alter only the mobs casting range right?
https://github.com/DarkstarProject/darkstar/pull/3608/files#diff-7fd2c48b6004c4fc5a400cdb8c98baa5R195 appears to be the part for mobs. Currently set to 28.5